Bottom: 91c6c7f4eeb156f05c173d357af069df7f62ad1e
Top:    5bff227459845a8d4712aa1ea6a59534ab1bbf62
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Re-enable board core emission


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 48ddf29..47499ce 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -152,8 +152,7 @@ step_do_export (HID_Attr_Val * options)
   if (filename == NULL)
     filename = "pcb-out.step";
 
-//  board_outline_list = object3d_from_board_outline ();
-  board_outline_list = NULL;
+  board_outline_list = object3d_from_board_outline ();
 
   board_outline = board_outline_poly (true);
   piece = board_outline;
@@ -199,11 +198,11 @@ step_do_export (HID_Attr_Val * options)
         poly_DelContour (&curc);
       }
 
-//    mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
-//    board_outline_list = g_list_concat (board_outline_list, mask_objects);
+    mask_objects = object3d_from_soldermask_within_area (piece, TOP_SIDE);
+    board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
-//    mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
-//    board_outline_list = g_list_concat (board_outline_list, mask_objects);
+    mask_objects = object3d_from_soldermask_within_area (piece, BOTTOM_SIDE);
+    board_outline_list = g_list_concat (board_outline_list, mask_objects);
 
     copper_layer_objects = object3d_from_copper_layers_within_area (piece);
     board_outline_list = g_list_concat (board_outline_list, copper_layer_objects);
